reV/supply_curve/supply_curve.py

```diff
@@ -279,7 +315,7 @@ def _merge_sc_trans_tables(cls, sc_points, trans_table,
 
         Returns
         -------
-        trans_table : pd.DataFrame
+        trans_sc_table : pd.DataFrame
             Updated table mapping supply curve points to transmission features.
             This is performed by merging left with trans_table, so there may be
             rows with nan sc_gid.
```

Comment:
what?? why would there be nan sc gid? When you say "merge left with trans_table" (ambiguous!), you mean the trans_table is the left table and how=left?

Reply:
I also think we should move the call `self._check_sc_trans_table(self._sc_points, trans_table)` into the `_merge_sc_trans_tables()` method

Reply:
hmm, good point,  that specific code is years old. I'll double check

sounds good on the check, i'll move that